sklearn/linear_model/stochastic_gradient.py

```diff
@@ -73,6 +76,13 @@ def __init__(self, loss, penalty='l2', alpha=0.0001, C=1.0,
         self.warm_start = warm_start
         self.average = average
 
+        if n_iter is not None:
+            warnings.warn("n_iter parameter is deprecated and will be removed"
```

Comment:
It's better to be very explicit: `"n_iter parameter is deprecated in 0.17 and will be removed in 0.19. ..."`